cEP-0005.md

```diff
@@ -124,6 +124,17 @@ class RedundancyBear(LocalBear):
             yield Result.from_values(aspect=aspect, ...)
 ```
 
+Before moving any further, it is quite necessary to have a clear-cut definition of both aspects and tastes, to avoid any kind of discrepancies and chaos in the future.
+
+## aspects
+An aspect should just talk about a property of something, without qualifying it. It should be named something like `Length` rather than `TooLong`.
+Then it should be given the tastes. For e.g. aspect `Metadata.CommitMessage.Shortlog.Length` can have a taste like `max_shortlog_length`.
+
+## tastes
```

Comment:
Same as @sils mentioned for `##aspects` heading in https://github.com/coala/cEPs/pull/72#discussion_r113806973